test/C/src/federated/DistributedLoopedActionDecentralized.lf

```diff
@@ -26,22 +26,11 @@ reactor Receiver(take_a_break_after: int = 10, break_interval: time = 400 msec)
 
   reaction(in) {=
     tag_t current_tag = lf_tag();
-    lf_print("At tag (%lld, %u) received value %d with STP violation (%lld, %u).",
-      current_tag.time - lf_time_start(),
-      current_tag.microstep,
-      in->value,
-      current_tag.time - in->intended_tag.time,
-      current_tag.microstep - in->intended_tag.microstep
+    lf_print("Inside, at tag " PRINTF_TAG ", received value %d.",
```

Comment:
What does "inside" mean?

Reply:
Inside the hierarchical reactor, to distinguish it from the reaction at the top level, which also reads the input.